Update office-add-ins-community-call.md (#4408)
OfficeDev/office-js-docs-pr: docs/overview/office-add-ins-community-call.md

---
title: Microsoft Office Add-ins community call
description: The monthly Microsoft Office Add-ins community call agenda, schedule, and resources.
ms.date: 02/14/2024
ms.localizationpriority: medium
---

# Microsoft Office Add-ins community call

The Microsoft Office Add-ins community call is a monthly call where you can learn more about new features, development practices, and additional information about creating Office Add-ins. The community call occurs the second Wednesday of each month at 7:00 AM Pacific Time. You can download the calendar invite at <https://aka.ms/officeaddinscommunitycall>.

## Agenda for February 14, 2024 call

> This month we will be having the following agenda topics:

Reviewer: ```suggestion
This month, we have the following agenda topics:
```

> - Retirement of entity-based contextual Office add-ins
> - Chrome’s privacy sandbox and 3P cookies

Reviewer: ```suggestion
- Retirement of entity-based contextual Office Add-ins.
- Chrome’s privacy sandbox and third-party cookies.
```

> > In order for us to be able to better improve our community call experience and understand our audience fill out this survey: https://forms.office.com/r/YNDtW8wQ6U

Reviewer: ```suggestion
> For us to improve our community call experience and better understand our audience, please fill out this survey: https://forms.office.com/r/YNDtW8wQ6U
```
